warehouse/integrations/datalake/schema-repository/local.go

```diff
@@ -34,10 +34,7 @@ func (*LocalSchemaRepository) CreateSchema() (err error) {
 
 func (ls *LocalSchemaRepository) CreateTable(tableName string, columnMap map[string]string) (err error) {
 	// fetch schema from local db
-	schema, _, err := ls.FetchSchema(ls.warehouse)
-	if err != nil {
-		return err
-	}
+	schema, _, _ := ls.FetchSchema(ls.warehouse)
```

Comment:
Since FetchSchema always returns nil, ignoring it.

Reply:
Should we remove the err from `FetchSchema` then? We can re-introduce it once error is possible.